src/js/schemas/service-schema/Volumes.js

```diff
@@ -4,6 +4,11 @@ const Volumes = {
   properties: {
     localVolumes: {
       title: 'Persistent Local Volumes',
+      description: (
+        <span>
+        Specify a local volume or volumes to “pin” tasks and their associated data to the node they are first launched on. Tasks will be relaunched on that node if they terminate. <a href="https://dcos.io/docs/1.7/usage/storage/persistent-volume/">Learn more about creating stateful tasks with persistent volumes</a>.
```

Comment:
We need to indent this once further and add `target="_blank"` to the external link.